openghg/plotting/_timeseries.py

```diff
@@ -1,12 +1,16 @@
+from __future__ import annotations
 import logging
 from typing import List, Optional, Union, Dict, Tuple
 import plotly.graph_objects as go
 import numpy as np
 import base64
+from typing import TYPE_CHECKING
```

Comment:
Out of interest - what is this doing?

Reply:
`TYPE_CHECKING` is always False at runtime so you can do imports during type checking to allow mypy to understand the types correctly.